vyper/semantics/validation/annotation.py

```diff
@@ -55,12 +55,7 @@ def visit(self, node):
         super().visit(node)
 
     def visit_Attribute(self, node):
```

Comment:
I think we can just toss this function now

Reply:
Sadly removing just completely borks with:
`vyper.exceptions.StructureException: Cannot annotate: Attribute`
:/

Reply:
:thinking:

Reply:
Can you leave a note to that effect since I'm guessing it will also confuse people (us?) in the future

Reply:
lol gotcha, pushed